### ISSUE TITLE
Output /p:ReportAnalyzer=true under normal verbosity

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [dotnet/roslyn#55604]
Hello,

We're using `/p:ReportAnalyzer=true` a lot when working with analyzer performance and the output is very useful. 

Downside of this property is that it works only with `-verbosity:detailed` that also outputs many MsBuild and target steps that we don't need. It's polluting our logs with unneeded stuff and makes the results hard to find.

Can you please update `/p:ReportAnalyzer=true` to output the results to the console under `-verbosity:normal`?

If we explicitly asks for this parameter, we'd like to have it under normal conditions.

**Comment by Youssef1313** [comment]
This is probably why I was never able to get it working without a binary log and opened this docs issue https://github.com/dotnet/docs/issues/24089 😄

**Comment by jinujoseph** [comment]
cc @sharwell

**Comment by sharwell** [comment]
The detailed output logs do not [aggregate the results](https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/452), so even if this change is made users will need to continue using MSBuild Structured Log Viewer to get a high-level overview of analyzer performance in their solutions.

**Comment by CyrusNajmabadi** [comment]
Moving to compiler as this is a compiler flag.

**Comment by jaredpar** [comment]
@rainersigwald, @baronfel 

AFAICT we're just emitting these messages `LogMessageFromText`. Nothing about that I can see that would be tied to msbuild verbosity. Am I missing something?

**Comment by baronfel** [comment]
The Message can have an Importance that various loggers will treat differently based on Verbosity. The older Console Logger will log High importance Messages by default, and other Importances at more verbose levels. Terminal Logger will only ever show High Importance Messages, and that only at Detailed or Diagnostic verbosity levels.

**Comment by jaredpar** [comment]
We emit these messages at the same verbosity as normal compiler output.

**Comment by rainersigwald** [comment]
Yeah I think the only way to accomplish this goal would be to enlighten your message handler to detect and reprioritize these messages.

**Comment by jaredpar** [comment]
What im missing is these messages are output with same priority as warnings / errors. Why is only one showing up? Is there another level of filtering I'm missing?

I thought we did some error / warning matching but not sure if that's on our side or your side.

**Comment by rainersigwald** [comment]
It's on our side--`LogMessageFromText` routes to either our `LogMessage`, `LogWarning` or `LogError`, depending on a regex match.

**Comment by jaredpar** [comment]
Interesting, is there a reason those don't live within our build tasks? Basically why does MSBuild decide what a C# error is vs. the C# build task? Is that just a historical artifact?
